captum/attr/_core/deep_lift.py

```diff
@@ -427,14 +488,16 @@ def _register_hooks(self, module):
         self.forward_handles.append(pre_forward_handle)
         self.backward_handles.append(backward_handle)
 
-    def _remove_hooks(self):
+    def _remove_hooks(self) -> None:
         for forward_handle in self.forward_handles:
             forward_handle.remove()
         for backward_handle in self.backward_handles:
             backward_handle.remove()
 
-    def _pre_hook_main_model(self):
-        def pre_hook(module, baseline_inputs_add_args):
+    def _pre_hook_main_model(self) -> RemovableHandle:
+        def pre_hook(
+            module: Module, baseline_inputs_add_args: Tuple[Tensor, ...]
```

Comment:
Since the tuple includes additional arguments which could be anything, the type of both baseline_inputs_add_args and the return type should be just Tuple.